README.rst

```diff
@@ -44,6 +44,10 @@ DEMO
 ----
 GIF_.  Live_.
 
+Demo page
```

Comment:
Every heading is in Block letters. Change your heading to block letters. Lets see how it looks.